### ISSUE TITLE
Saved Capability Sets are being lost

### ISSUE THREAD

**Issue by toien44** [appium/appium-desktop#854]
## The problem

Appium Desktop v1.11.0 when you close the app you lose your saved capability set

**Comment by dpgraham** [comment]
@toien44 that's correct yes. My recommendation is to Save your desired capabilities before starting a session.

It is possible that we can always save the most recently used desired capabilities so that when you return to the session creation screen, it goes back to the caps that you most recently used.

I don't think I'll have time to prioritize this feature but I'd be happy to accept a pull request and help somebody get this feature into Appium Desktop.

**Comment by toien44** [comment]
Before I had a few saved in Saved Capability Set and it didn't delete them. So every time you close Appium Desktop it will delete that section?

**Comment by dpgraham** [comment]
If you save it, it will be there, you just have to re-select it when you return.

**Comment by robegort** [comment]
actually the saved capabilities aren't saved anymore. I've had numerous occasions where the saved capabilities are gone when I start appium Version 1.11.0 (1.11.0)

**Comment by NastyAlloys** [comment]
Same. Every time I restart Appium it asks me to send crash log and no saved capabilities are present.

**Comment by dpgraham** [comment]
@robegort @NastyAlloys are you saying _all_ saved capabilities are gone? So you're saving caps and then closing and returning to the app and nothing is there?

**Comment by dpgraham** [comment]
ie: there's nothing under `Saved Capabilities Sets`

**Comment by ranjith3037** [comment]
Same here. I had 11 saved capabilities yesterday on v1.10.0, today when it was updated to v1.11.0, I see 0 in my saved capabilities list

**Comment by dpgraham** [comment]
I'm terribly sorry that this happened to all of you. This is obviously a bug on the Appium Desktop side.

One more question: from 1.11.0 onwards, does saving the capabilities no longer work? Or did it lose everything prior to 1.11 but then saves them in the future?

I wish I had a solution for recovering the lost capability sets, but I don't know if that's possible.

**Comment by dpgraham** [comment]
Is this happening on Windows, Linux or Mac?

This didn't happen for me on Mac.

**Comment by toien44** [comment]
It started for my on a Mac. It doesn't happen every time. It started for my in 1.11.0

**Comment by halderpritam** [comment]
Facing the same thing over here. For me it happens on Mac

**Comment by robegort** [comment]
For me it happened on mac and started on 1.11.0 or 1.11.1

**Comment by dpgraham** [comment]
This should be resolved with 1.12 now.... let me know if anyone still experiences this.

And again, my apologies for the terrible inconvenience that this would have caused.

**Comment by surendra-savita-30060677** [comment]
I am still facing same issue in Appium 1.21.0-1 version. I am using mac. I saved capabilities but it is not showing under saved capabilities tab. Even saved capabilities set is disable means this option is showing greyed out. Can somebody provide me solution for the same?

**Comment by sanjaybhaithakur** [comment]
This is happening for me also. I tried to upgrade and downgrade appium, but still this doesn't resolve the issue.

**Comment by Nickolay-Sachno** [comment]
I'm also facing the same issue, I'm having Mac, Appium 1.20,1.22, 1.18

**Comment by mdowgo** [comment]
This use to work for me.  Had to change laptops and I can't save my desired capabilities anymore on the new laptop. 
Mac 10.15.7
Appium 1.20.2

**Comment by mdowgo** [comment]
@jlipps, I attempted to use the latest version 2022.2.1.  However, there is an issue that doesn't make sense.  Previously I wasn't using the `platformName` parameter and it was working fine.  It complained that it needed the parameter so I added it, but it keeps asserting an error that it is not present. 

![2022-04-11_10-55-43](https://user-images.githubusercontent.com/29473191/162800484-4c8fcad3-dc91-42c0-a620-0e25f0e6c806.png)
